### DISCUSSION TITLE
Feedback Request: Course Lesson

### DISCUSSION THREAD

**Discussion by DavidLandup0** [keras-team/keras-cv#649] [Show and tell]
Hey! I've been writing a course on practical deep learning for CV since the beginning of the year. I opted for Keras, as it's IMO, the best library to showcase practical, simple, easy-to-follow examples and produce great results, with minimal boilerplate and fluff.

A good portion of the course is tied to tools of the trade, and covers `tf-keras-vis`, `tfds`, what-if, etc. and a huge emphasis is put on developing skills to be a CV engineer (finding, understanding and implementing papers, finding and using tools, optimizing training pipelines, etc.). One of the tools covered in the course is, naturally, KerasCV!

Since it's still in development, I left some space to update the course as KerasCV develops as well. Here's a free open lesson written as an introduction to the lib. It would really mean a lot if any feedback was given for it! 

- https://stackabuse.com/courses/practical-deep-learning-for-computer-vision-with-python/lessons/keras-cv/

@LukeWood @bhack

**Comment by jbischof** [comment]
I read the book version of your course @DavidLandup0 and loved it! We've made a lot of changes to KerasCV since you wrote it and would love to connect some time.
